### PR TITLE
enh(userstatus): set user status to 'In a meeting' if calendar is busy

### PR DESCRIPTION
**Pull request by miaulalala**
### 🖼️ Screenshots

image

### CODE REVIEW

**Review by GretaD — APPROVED** [review]

File: user_manual/groupware/calendar.rst [hunk 1]
```diff
@@ -265,6 +265,13 @@ The ``Empty trash bin`` buttons will wipe all trash bin contents in one step.
 
 .. _calendar-working-hours:
 
+Automated User Status
+~~~~~~~~~~~~~~~~~~~~~
+
+When you have a calendar event scheduled that has a "BUSY" status, your user status will be automatically set to "In a meeting" unless you have set yourself to "Do Not Disturb" or "Invisible".
+You can overwrite the status with a custom message any time, or set your calendar events to "FREE".
+Calendars that are TRANSPARENT will be ignored.
```
Comment (tcitworld) [inline user_manual/groupware/calendar.rst:273]: Don't you mean events?

Comment (miaulalala) [reply user_manual/groupware/calendar.rst:273]: Well no there's apparently the option to set a calendar to TRANSPARENT as well

Comment (miaulalala) [reply user_manual/groupware/calendar.rst:273]: `schedule-calendar-transp`

Comment (tcitworld) [reply user_manual/groupware/calendar.rst:273]: I'd drop the all-caps to distinguish between icalendar properties and caldav ones.

**Pull request merged**